APMrover2/radio.cpp

```diff
@@ -9,10 +9,12 @@ void Rover::set_control_channels(void)
     channel_steer    = RC_Channels::rc_channel(rcmap.roll()-1);
     channel_throttle = RC_Channels::rc_channel(rcmap.throttle()-1);
     channel_aux      = RC_Channels::rc_channel(g.aux_channel-1);
+    channel_lateral  = RC_Channels::rc_channel(rcmap.yaw()-1);
 
     // set rc channel ranges
     channel_steer->set_angle(SERVO_MAX);
     channel_throttle->set_angle(100);
+    channel_lateral->set_angle(SERVO_MAX);
```

Comment:
This should probably be -100 <-> 100 - feels more like a throttle value than a steering value.